core/tests/protractor_desktop/publicationAndLibrary.js

```diff
@@ -171,6 +174,32 @@ describe('Library index page', function() {
     users.logout();
   });
 
+  it('should save exploration to play later list', function() {
+    var EXPLORATION_FRACTION = 'fraction';
+    var CATEGORY_ARCHITECTURE = 'Architecture';
+    var LANGUAGE_ENGLISH = 'English';
+    var EXPLORATION_OBJECTIVE = 'hold the light of two trees';
+    learnerDashboardPage = new LearnerDashboardPage.LearnerDashboardPage();
+
+    users.createUser(
+      'feanor@publicationAndLibrary.com', 'feanorPublicationAndLibrary');
+    users.login('feanor@publicationAndLibrary.com');
+    workflow.createAndPublishExploration(
+      EXPLORATION_FRACTION, CATEGORY_ARCHITECTURE,
+      EXPLORATION_OBJECTIVE, LANGUAGE_ENGLISH);
+    users.logout();
+    users.createUser(
+      'celebrimor@publicationAndLibrary.com', 'celebriorPublicationAndLibrary');
```

Comment:
Maybe we can name these users as creator and learner? For readability :)

Reply:
Done!